jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -218,7 +218,8 @@ public void createDefaultTemplates(final String jvmName, Group parentGroup) {
             try {
                 ResourceTemplateMetaData metaData = resourceService.getMetaData(metaDataStr);
                 if (metaData.getEntity().getDeployToJvms()) {
-                    final String template = resourceService.getAppTemplate(groupName, metaData.getEntity().getTarget(),
+                    String applicationName = applicationService.findApplications(groupPersistenceService.getGroup(groupName).getId()).iterator().next().getName();

```

Comment:
You are assigning the first app, what if the group has multiple apps, will this work?